src/Toggle/CreateMarketingInformationForToggle.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: Volodymyr
+ * Date: 10/08/2015
+ * Time: 09:52
+ */
+
+namespace Clearbooks\Labs\Toggle;
+
+
+use Clearbooks\Labs\Toggle\Entity\CreateMarketingInformationRequest;
+use Clearbooks\Labs\Toggle\Gateway\MarketableToggleGateway;
+
+class CreateMarketingInformationForToggle
+{
+    /**
+     * @var MarketableToggleGateway
+     */
+    private $gateway;
+
+    /**
+     * CreateMarketingInformationForToggle constructor.
+     * @param MarketableToggleGateway $gateway
+     */
+    public function __construct( MarketableToggleGateway $gateway )
+    {
+        $this->gateway = $gateway;
+    }
+
+    /**
+     * @param CreateMarketingInformationRequest $request
+     */
+    public function execute( CreateMarketingInformationRequest $request )
+    {
+
+        $this->gateway->setMarketingInformationForToggle( $request );
```

Comment:
I don't think I would pass the request straight to the gateway, The gateway shouldn't need to know about how the use case is called. I think two parameters, 1) the toggle id and 2) an array in the format of [ 'column_name' => 'value' ]